index.html

```diff
@@ -6,7 +6,13 @@
     <title>Todo App-Vanilla JS</title>
 </head>
 <body>
-    
+    <h1>Add Todos</h1>
+    <div id="addTodoContainer">
+        <input type="text" id="addTodoInputField">
```

Comment:
`addTodo` prefix is redundant as we know the scope of the project
```suggestion
        <input type="text" id="input-field">
```

Reply:
okay bhai.